### PR TITLE
Improve the docstring of close

### PR DESCRIPTION
**Pull request by Make42**
## Related Issue \ discussion
https://app.slack.com/client/TT9ATQXJ5/CTK20V944/thread/CTK20V944-1676556098.205589

## Patch Description
It was not clear to me, how close and mark_completed work.

## Testing Instructions
Make sure that my documentation is correct. Not software test needed.

### CODE REVIEW

File: clearml/task.py [hunk 1]
```diff
@@ -1724,8 +1724,13 @@ def reset(self, set_started_on_success=False, force=False):
 
     def close(self):
         """
-        Closes the current Task and changes its status to completed.
+        Closes the current Task and changes its status to "Completed".
         Enables you to manually shutdown the task.
+        It does not terminate the current Python process, in contrast to :meth:`Task.mark_completed`.
```
Comment (jkhenning) [inline clearml/task.py:1729]: Perhaps rephrase to "This method will not terminate..."

File: clearml/task.py [hunk 2]
```diff
@@ -1724,8 +1724,13 @@ def reset(self, set_started_on_success=False, force=False):
...
+        
+        After having :meth:`Task.close`d a task, the respective object cannot be used anymore and
+        methods like :meth:`Task.connect` or :meth:`Task.connect_configuration` will throw a `ValueError`.
+        However, you can "get the task back" by using methods like :meth:`Task.get_task`.
```
Comment (jkhenning) [inline clearml/task.py:1733]: Perhaps rephrase to "In order to obtain an object representing the task again, use methods like ..."

File: clearml/task.py (hunk 1 above)
Comment (Make42) [reply clearml/task.py]: I included your proposal.

File: clearml/task.py (hunk 2 above)
Comment (Make42) [reply clearml/task.py]: I included your proposal.

**Pull request merged**